orc8r/cloud/go/syncstore/cache.go

```diff
@@ -0,0 +1,149 @@
+/*
+ Copyright 2020 The Magma Authors.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree.
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package syncstore
+
+import (
+	"database/sql"
+	"fmt"
+
+	"magma/orc8r/cloud/go/sqorc"
+
+	"github.com/Masterminds/squirrel"
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+)
+
+type cacheWriter struct {
+	network        string
+	id             string
```

Comment:
Throughout: I think "id" is actually the "type" of the cached objects? Let's use "type" if so

Reply:
I thought the `id` here is the unique identifier for each cache writer (which is also conveniently used as the name for the temporary SQL table) @hcgatewood